1281.subtract-the-product-and-sum-of-digits-of-an-Integer.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/subtract-the-product-and-sum-of-digits-of-an-integer/
+ */
+
+/**
+ * @param {number} n
+ * @return {number}
+ */
+
+var subtractProductAndSum = function(n) {
+    
+    let stringNumber = n.toString();
```

Comment:
```suggestion
    const stringNumber = n.toString();
```
Const is better here.

Reply:
I corrected it